context/selection-of-signal-for-I.md

```diff
@@ -0,0 +1,165 @@
+* **[Average is considered to be a proxy for long run marginal](https://github.com/Green-Software-Foundation/sci/pull/353#issuecomment-1432307783)**, it’s easier to calculate and is correlated with long run marginal. Furthermore, AER strongly correlates with curtailment of renewables. Average emission factors perform poorly for estimating the short-term impact of interventions. 
+
+
+### How do we define _better _in terms of which signal is the _better _choice?
+
+**Andrew Woosnam**: Better could be defined as:
+* Avoiding emissions right now.
+* Working toward the “ideal grid of the future” (e.g. easily served by new solar)
+
+
+### What’s the scientific consensus regarding which has better impact in terms of grid decarbonisation?
+
+There is scientific consensus ([source](https://www.cell.com/iscience/fulltext/S2589-0042(22)00185-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2589004222001857%3Fshowall%3Dtrue)) that LRMER are the best signal to decarbonise the grid on a longer term. AER are strong proxies for LRMER, while SRMER omits long-term impacts of interventions. There can be other signals, such as renewable power potential curves, residual load curves, etc. that should be investigated. 
+
+
+### Are there other signals other than short run marginal, long run marginal and average?
+
+* **Asim Hussain**: There was mention in the past of “deaths per kWh”, which although won’t align with the rest of the SCI is an interesting view on the idea of signals.
+* **Ciril Wakounig:** Renewable potential profiles, Henry mentioned net/residual load signals
+
+
+### Independent of the question of SMER, LMER or AER, does the value of I need to be hourly?
+
+* If you are implementing time-shifting of compute hourly data is important.
+* If you are implementing some form of regional shifting, or making a decision where to place your compute long term if shifting real-time isn’t an option, then yearly averages for I might be reasonable..
+
+
+### Will there be global standards for these signals?
+
+**Ciril Wakounig**: Overall, out of the three emission factors, average is very trivial to calculate, as it is a weighted average of generation and related emission factors. Most policy/regulatory/governmental communication and regulation is based on average emission factors, without a general standard (I guess because it is implied). 
+
+For marginal (short- and long-term) it is very difficult to develop standards, since they are both based on assumptions and statistical models that approximate reality. Contrary to the average, it is not possible to compare marginals to what happened in reality and thus it is quite difficult (I assume not impossible) to develop a standard. For instance, the average can easily be verified based on the hourly power mix that is published by TSOs.
+

```

Comment:
I added a short summary section with the two points of view which emerged during the last two weeks of discussions on the topic.
```suggestion

## Summary

The foundation's members are debating which signal is best to use when calculating carbon intensity. Two main perspectives are being taken into account. The first point of view encourages the use of two signals, one for reporting (e.g., AER) and the other for real-time operational decision-making (e.g., SMER or energy fraction). This approach has the advantage of simplicity and comparability between different projects. The second point of view advises leaving the freedom to employ various signals in accordance with the particular use cases and considering the advantages and disadvantages of each choice. These conversations demonstrate a careful examination of the intricacies involved in determining carbon intensity and emphasize the significance of selecting the most appropriate strategy for each particular circumstance.
```